### PR TITLE
Add funding credits and purpose

### LINKED ISSUE #7: Write funding & purpose

### PR DESCRIPTION
**Pull request by lindsayplatt**
Fixes #7. Kept default purpose because it matches what we want, but added our funding credits.

### CODE REVIEW

File: in_text/text_data_release.yml [hunk 1]
```diff
@@ -33,6 +33,12 @@ abstract: >-
   10. "lake_temp_metrics_glm_nldas.csv": annual lake temperature metrics for the contemporary period derived from daily predictions generated by GLM (12,688 lakes)
   11. "lake_temp_metrics_glm_gcm.csv": annual lake temperature metrics for future climate scenarios derived from daily predictions generated by GLM (11,715 lakes)
   
+  This work was completed with funding support from the Midwest Climate Adaptation Science Center (MW CASC) 
+  and as part of the USGS project on Predictive Understanding of Multiscale Processes, an element of the
```
Comment (padilla410) [inline in_text/text_data_release.yml:37]:
```suggestion
  and as part of the USGS project on Predictive Understanding of Multiscale Processes (PUMP), an element of the 
```

File: in_text/text_data_release.yml [hunk 2]
```diff
@@ -136,8 +142,8 @@ entities:
 
 
 purpose: Fisheries biology, limnological research, and climate science.
-start-date: 19800401
-end-date: 20191231
+start-date: 19800101
+end-date: 20991231
```
Comment (padilla410) [inline in_text/text_data_release.yml:146]: Just double checking that our predictions go out to 2099

File: in_text/text_data_release.yml [hunk 3]
```diff
@@ -173,8 +179,11 @@ metadata-date: !expr format(Sys.time(),'%Y%m%d')
 
 accur-test: No formal attribute accuracy tests were conducted.
 funding-credits: >-
-  This study was funded by the [DETAILS HERE]
-  This research used resources of the Core Science Analytics and Synthesis Advanced Research Computing program at the U.S. Geological Survey.
+  This research was funded by the Midwest Climate Adaptation Science Center (MW CASC) and the USGS project on 
+  Predictive Understanding of Multiscale Processes, an element of the Integrated Water Prediction Program,
```
Comment (padilla410) [inline in_text/text_data_release.yml:183]:
```suggestion
  Predictive Understanding of Multiscale Processes, an element of the Integrated Water Prediction Program (PUMP), 
```

**Review by padilla410 — APPROVED** [review]
Approve after inline comments are addressed!

File: in_text/text_data_release.yml (hunk 2 above)
Comment (lindsayplatt) [reply in_text/text_data_release.yml:146]: Yes, that's the late century climate projections

**Pull request merged**